### PR TITLE
Feature/methodology buttons

### PR DESCRIPTION
**Pull request by Bluesmile82**
This PR solves three distinct issues:

- **Show little countries as circles in the map**
    - Show the circles only at low levels of zoom:
      - Now we have different layers in the paths JSON: COUNTRIES, ISLANDS and POINTS

- **Show Europe as EU28 in the map**
  - Update paths JSON.
  - Little hack for reordering paths when we hover over EU28 on the list so we can see the shape highlighted.

- **Add methodology info buttons**
  - Add new info buttons and update the slugs for the modal metadata.

**Extra:** Fix layout (the margins were bigger than in the designs)

image
image

### CODE REVIEW

**Comment by simaob** [comment]
I think there's a mismatch in the info window of the NDC Content Overview. Looks like it's displaying the NDC Quantifications.

@juancarlosalonso can you check the small islands states pages maybe? When you can zoom in the circles feel a bit weird :/

**Comment by simaob** [comment]
What I mean by it: 
screen shot 2017-11-28 at 13 21 02

**Comment by Bluesmile82** [comment]
Now we show the circles only at low levels of zoom

File: app/javascript/app/components/countries-select/countries-select-selectors.js [hunk 1]
```diff
@@ -76,32 +77,45 @@ const semiActiveCountryStyles = {
...
+    });
 
-      const isInFilter = query ? nameUpper.includes(query) : false;
-      const style = isInFilter ? semiActiveCountryStyles : countryStyles;
-      return {
-        ...path,
-        style
-      };
-    })
+    // reorder map paths to show EU geometry if selected
```
Comment (edbrett) [inline app/javascript/app/components/countries-select/countries-select-selectors.js]: 🎖

File: app/javascript/app/components/country-ndc-overview/country-ndc-overview-styles.scss [hunk 2]
```diff
@@ -21,9 +21,13 @@
 .actions {
   @extend %grid;
 
-  @include msGridColumns(1fr, 2fr, 3fr);
+  @include msGridColumns(1fr, 20fr, 32fr);
```
Comment (edbrett) [inline app/javascript/app/components/country-ndc-overview/country-ndc-overview-styles.scss]: This seems a little crazy.

**Pull request merged**